src/core/reference.js

```diff
@@ -129,14 +257,14 @@
  * <code>
  * for (let i = 0; i < 9; i++) {
  *   console.log(i);
- *   // more statements
  * }
  * </code>
  * </div>
  */
 
 /**
- * Iterates over the enumerable properties of an object, in arbitrary order.
+ * From <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in">the MDN entry</a>:
+ * Iterates over the enumerable properties of an <a href="#/p5/object">object</a>, in arbitrary order.
```

Comment:
I agree that `for...in` and `for...of` here may be better ommited as the description here taken from MDN are pretty opaque, unless we write our own descriptions of them.

Reply:
I think also since for...in is almost never what one wants, maybe we should avoid it. Maybe an example of using `Object.keys();` and a `for` loop somewhere would be helpful though?